homework-submission/my-project/src/TodoItem.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const TodoItem = props =>
+  <div className="todo-item">
+    <p1>{props.description}</p1>
+    <p2>{props.deadlineDate}</p2>
```

Comment:
There is nothing called p1 or p2, i believe you want to create a paragraph then you have to use p tag.